e2e/playwright/point-click.spec.ts

```diff
@@ -2434,6 +2477,41 @@ extrude001 = extrude(sketch001, length = 40)
       await scene.expectPixelColor([49, 49, 49], testPoint, 15)
     })
 
+    await test.step('Edit shell via feature tree selection works', async () => {
```

Comment:
I'm not sure I see a distinction between what this test is covering and the test earlier in this file covers, what am I missing?

Reply:
@franknoirot one is working with shell on cap faces while this one is working with wall faces. Which have slightly different code paths and for a moment I had one working but not the other

Reply:
Ah I missed that sorry

Reply:
No worries!